Specs/Number/Japanese/NumberModel.json

```diff
@@ -1,652 +1,915 @@
 [
   {
-    "Input": "西九条",
-    "Results": []
-  },
-  {
-    "Input": "一匹",
+    "Input": "答えは１９２です。",
```

Comment:
Same as in Arabic, please don't modify/remove the existing test specs unless they really don't apply.
Let's follow the same approach discussed before.